pkg/watcher/reconciler/pipelinerun/controller.go

```diff
@@ -43,6 +48,7 @@ func NewControllerWithConfig(ctx context.Context, client pb.ResultsClient, cfg *
 	}
 
 	impl := controller.NewImpl(c, logger, "PipelineRunResultsWatcher")
+	c.enqueue = impl.EnqueueKeyAfter
```

Comment:
Any reason to prefer `EnqueueKeyAfter` instead of `EnqueueAfter` which can just take the whole object?